src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -356,8 +356,8 @@ protected override int CreateNavInfo(SYMBOL_DESCRIPTION_NODE[] rgSymbolNodes, ui
                 }
             }
 
-            var namespaceName = new StringBuilder();
-            var className = new StringBuilder();
+            var namespaceName = SharedPools.Default<StringBuilder>().AllocateAndClear();
```

Comment:
No. IIRC, there was long discussion about ```ObjectPool<T>``` having some IDisposable pattern and the decision that it shouldn't. I don't recall the details.